Signal/src/call/NonCallKitCallUIAdaptee.swift

```diff
@@ -43,23 +43,32 @@ class NonCallKitCallUIAdaptee: CallUIAdaptee {
         }
     }
 
-    public func reportMissedCall(_ call: SignalCall, callerName: String) {
+    internal func reportMissedCall(_ call: SignalCall, callerName: String) {
```

Comment:
Huh, I'm surprised these should be internal, since they're implementing public methods in the protocol.